e2e/test/scenarios/visualizations/drillthroughs/chart_drill.cy.spec.js

```diff
@@ -640,6 +640,131 @@ describe("scenarios > visualizations > drillthroughs > chart drill", () => {
       cy.findByText("Doohickey").should("not.exist");
     });
   });
+
+  it("should display proper drills on chart click for line chart", () => {
+    cy.createQuestion(
+      {
+        name: "Line chart drills",
+        query: {
+          "source-table": ORDERS_ID,
+          aggregation: [["count"]],
+          breakout: [
+            [
+              "field",
+              PRODUCTS.CREATED_AT,
+              { "source-field": ORDERS.PRODUCT_ID, "temporal-unit": "month" },
+            ],
+            ["field", PRODUCTS.CATEGORY, { "source-field": ORDERS.PRODUCT_ID }],
+          ],
+        },
+        display: "line",
+      },
+      { visitQuestion: true },
+    );
+
+    cy.get(".LineAreaBarChart").get(".dot").first().click({ force: true });
+    popover().within(() => {
+      cy.findByText(`See these Orders`).should("be.visible");
+
+      cy.findByText(`See this month by week`).should("be.visible");
+
+      cy.findByText(`Break out by…`).should("be.visible");
+      cy.findByText(`Automatic insights…`).should("be.visible");
+
+      cy.findByText(`>`).should("be.visible");
+      cy.findByText(`<`).should("be.visible");
+      cy.findByText(`=`).should("be.visible");
+      cy.findByText(`≠`).should("be.visible");
+    });
```

Comment:
nit: we should prefer double quotes when we don't need template strings

Reply:
https://github.com/metabase/metabase/pull/32471 @kulyk